src/NRules/NRules.Testing/RuleTestFixture.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using NRules.Fluent;
+using NRules.Fluent.Dsl;
+using NRules.RuleModel;
+
+namespace NRules.Testing;
+
+public class RuleTestFixture : IRuleTestFixture
+{
+    private readonly Lazy<ISession> _lazySession;
+    private readonly IRuleActivator _ruleActivator;
+    private readonly RuleCompiler _compiler;
+    private readonly RepositorySetup _setup;
+
+    public RuleTestFixture(IRuleActivator ruleActivator, RuleCompiler compiler, IRuleAsseter asseter)
```

Comment:
@gritcsenko typo in iface name and parameter name: asserter

Reply:
Fixed, thanks